src/transaction_builder.js

```diff
@@ -367,6 +384,9 @@ TransactionBuilder.prototype.__build = function (allowIncomplete) {
   if (!allowIncomplete) {
     if (!this.tx.ins.length) throw new Error('Transaction has no inputs')
     if (!this.tx.outs.length) throw new Error('Transaction has no outputs')
+
+    // do not rely on this, its merely a last resort
+    if (this.__hasAbsurdFee()) throw new Error('Transaction has absurd fees')
```

Comment:
@afk11 if you're doing 150kb transactions 120 satoshis/byte,  only then,  will you hit this limit.
With a constant `TransactionBuilder.ABSURD_FEERATE`,  a user could change that to by-pass these limits themselves,  as they will be a *very* select few people.